options/options.go

```diff
@@ -252,8 +242,6 @@ func New(appName, versionStr, gitCommit, usageStr string, parsePositionalArgsAsU
 
 	opts.parser.UnknownOptionHandler = opts.handleUnknownOption
 
-	opts.URI.AddKnownURIParameters(KnownURIOptionsReplicaSet)
```

Comment:
After removing all the callers, we should be able to remove the function from `options.go`, right?
```
func (uri *URI) AddKnownURIParameters(uriFieldNames []string) {
```

Reply:
Yes! Good point! Consider it done!